tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclJsonUtilTest.xtend

```diff
@@ -0,0 +1,41 @@
+package org.testeditor.tcl.dsl.jvmmodel
+
+import javax.inject.Inject
+import org.junit.Test
+import org.testeditor.dsl.common.util.JvmTypeReferenceUtil
+import org.testeditor.tcl.dsl.tests.AbstractTclTest
+import org.junit.Before
+import org.eclipse.emf.ecore.resource.Resource
+
+class TclJsonUtilTest  extends AbstractTclTest {
+	@Inject TclJsonUtil jsonUtil // class under test
+	
+	@Inject JvmTypeReferenceUtil typeReferenceUtil
+	
+	@Before def void setupTypeReferenceUtil() {
+		typeReferenceUtil.initWith(null as Resource)
+	}
+	
+	@Test def void testJsonArrayTypeDetection() {
+		// given
+		val jsonArrayType = typeReferenceUtil.jsonArrayJvmTypeReference
+		
+		// when
+		val isJsonType = jsonUtil.isJsonType(jsonArrayType)
+		
+		// then
+		isJsonType.assertTrue		
+	}
+
+	@Test def void testStringNotJsonTypeDetection() {
```

Comment:
That's a new style. In the other test we only allowed `@Inject` to be on the same line, all other annotations were kept on a separate line.